chord_metadata_service/restapi/authz_middleware.py

```diff
@@ -13,9 +13,14 @@ def __init__(self, get_response):
 
     def __call__(self, request):
         if request.path == "/api/phenopackets" and request.method == "GET":
+            tokens = {}
+            for header in request.headers:
+                if header.startswith(("X-CANDIG-LOCAL", "X-CANDIG-DAC", "X-CANDIG-FED", "X-CANDIG-EXT")):
```

Comment:
What do you all think about explicitly adding a hyphen at the end of each header like 

```
header.startswith(("X-CANDIG-LOCAL-", "X-CANDIG-DAC-", "X-CANDIG-FED-", "X-CANDIG-EXT-")):
```

My thinking is that without hyphen they will match words as well and may limit naming choices. Not a big deal right now though and this can be adjusted later on too.

Reply:
I don't mind changing it so that it checks for the hyphen at the end as well.

Reply:
Alright, I have added a hyphen at the end of the terms